SherpaFrontEnd.Tests/Shared/Modals/CreateTeamOffcanvasTest.cs

```diff
@@ -53,6 +53,16 @@ public void ShouldRenderAllInputsAndButtons()
         Assert.NotNull(continueButton);
         Assert.NotNull(cancelButton);
     }
+    
+    [Fact]
+    public void ShouldRenderOnTheRightSideOfTheScreen()
+    {
+        var component = _testCtx.RenderComponent<CreateTeamOffcanvas>();
+
+        var createTeamForm = component.FindAll("div").FirstOrDefault(element => element.ClassList.Contains("offcanvas-end"));
+        Assert.NotNull(createTeamForm);
+
+    }
```

Comment:
Being something purely related with the styles and also being the class given to us by bootstrap, I wouldn't create this test. I think it could become flaky with ease.

Reply:
That makes sense. I tried to use the test as the trigger to at least have a conversation about it.